tests/test_browsers_iradio.py

```diff
@@ -170,7 +170,7 @@ def cb(data):
         assert all(ret)
         assert self.received, "No stations received from %s" % url
         assert len(self.received) > 100
-        # TODO: some more targetted tests
+        # TODO: some more targeted tests
```

Comment:
This one is actually debatable / locale-dependent ([wiktionary targetted](https://en.wiktionary.org/wiki/targetted)) but seems reasonable to use the more common :+1: